examples/magento-graphcms/locales/en.po

```diff
@@ -469,9 +463,6 @@ msgstr "Shipping address"
 msgid "Shipping method"
 msgstr "Shipping method"
 
-msgid "No shipping methods available"
-msgstr "No shipping methods available"
```

Comment:
This string has accidentally became untranslatable, can you make it translatable again (ShippingMethodForm)?

Reply:
I've added the translation